recipe/meta.yaml

```diff
@@ -30,25 +33,28 @@ requirements:
     - python
     # https://github.com/huggingface/accelerate/blob/665d5180fcc01d5700f7a9aa3f9bdb75c6055dce/src/accelerate/utils/torch_xla.py#L18
     - setuptools
-    - numpy >=1.17
+    - numpy 1.17,<2.0.0
```

Comment:
```suggestion
    - numpy >=1.17,<2.0.0
```

Reply:
Fixed this constraint in https://github.com/AnacondaRecipes/accelerate-feedstock/pull/5/commits/906864b0cbeafa9ad0f0c8c8feb5b557386b5b47